src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -106,83 +108,80 @@ private static GitPullRequest CreatePullRequest(string sourceBranch, string targ
             return newestBuild;
         }
 
-        private static async Task<Build> QueueValidationBuildAsync(string branchName)
+        // Similar to: https://devdiv.visualstudio.com/DevDiv/_git/PostBuildSteps#path=%2Fsrc%2FSubmitPullRequest%2FProgram.cs&version=GBmaster&_a=contents
+        private static async Task QueueBuildPolicy(GitPullRequest pullRequest, string buildPolicy)
         {
-            var buildClient = ProjectCollection.GetClient<BuildHttpClient>();
+            WaitForMergeComplete(pullRequest);
```

Comment:
Seems weird to have a void blocking method inside an async one.

Reply:
Yeah, but there is no async API to get this information

Reply:
Not "seems"..."it is"... 😸